android/src/main/java/com/pspdfkit/react/ReactPdfViewManager.java

```diff
@@ -114,6 +148,44 @@ public void receiveCommand(PdfView root, int commandId, @Nullable ReadableArray
             case COMMAND_SAVE_CURRENT_DOCUMENT:
                 root.saveCurrentDocument();
                 break;
+            case COMMAND_GET_ANNOTATIONS:
+                if (args != null) {
+                    final int requestId = args.getInt(0);
+                    Disposable annotationDisposable = root.getAnnotations(args.getInt(1), args.getString(2))
```

Comment:
What do you need the `annotationDisposable` for? I assume you could remove this local variable. Does `ReactPdfViewManager` have some lifecycle we could tap into to dispose ongoing subscriptions at some point?